go/base/static/js/test/local.test.js

```diff
@@ -0,0 +1,36 @@
+describe("go.local", function() {
+  afterEach(function() {
+    localStorage.clear();
+  });
+
+  describe(".set", function() {
+    it("should stringify and set values", function() {
+      assert(!('foo' in localStorage));
+      go.local.set('foo', {bar: 23});
+      assert.deepEqual(JSON.parse(localStorage.foo), {bar: 23});
+    });
+  });
+
+  describe(".get", function() {
+    it("should return null for non-existent items", function() {
+      assert(!('foo' in localStorage));
+      localStorage.bar = JSON.stringify({baz: 21});
+
+      assert.strictEqual(go.local.get('foo'), null);
+      assert.notStrictEqual(go.local.get('bar'), null);
+    });
+
+    it("should support default values", function() {
+      assert(!('foo' in localStorage));
+      localStorage.bar = JSON.stringify({baz: 21});
+
+      assert.strictEqual(go.local.get('foo', 23), 23);
```

Comment:
Can you clarify this one for me, please?
I'm not sure about this `assert`: it looks like the setting of `foo` happens in a different `describe` block?